doc/src/developer_guide.rst

```diff
@@ -224,6 +219,11 @@ The families of schemes follow a naming convention
     :exclude-members: _abc_cache, _abc_negative_cache,
                       _abc_negative_cache_version, _abc_registry
 
+Note that :code:`DualDenseRKPseudoStepper` includes families of
```

Comment:
I don't think we should have the :code: here. We seem to reserve that just for function names.

Reply:
Removed.